### PR TITLE
Write Description: section

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,12 @@ groonga-log
 
 ## Description
 
-TODO:
+Groonga-log is a collection of library and tools to
+process [Groonga](http://groonga.org/)'s log.
+
+Groonga's log is logged when --log-path option is specified.  You can
```
Comment (kou) [inline README.md]:
```text
`--log-path`
```

File: groonga-log.gemspec [hunk 2]
```diff
@@ -20,13 +20,23 @@ lib = File.expand_path('../lib', __FILE__)
...
+end
+
 Gem::Specification.new do |spec|
   spec.name          = "groonga-log"
   spec.version       = Groonga::Log::VERSION
 
   spec.authors       = ["Horimoto Yasuhiro"]
   spec.email         = ["horimoto@clear-code.com"]
 
+  readme = File.read("README.md")
```
Comment (kou) [inline groonga-log.gemspec]: `File.read("README.md", :encoding => "UTF-8")`

**Comment by kenhys** [comment]
Thanks @kou 

* Fixed markup to `--log-path`.
* Fixed to use `File.read("README.md", :encoding => "UTF-8")`

**Comment by kou** [comment]
Merged.

**Pull request merged**